Rakefile

```diff
@@ -2,5 +2,6 @@
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
 require_relative 'config/application'
+require 'single_test/tasks'
```

Comment:
- [ ] `require 'single_test/tasks'` is giving me an error and not letting me migrate, I removed it and was able to migrate, maybe you have missing gems or this line isn't doing anything? please check it.

Reply:
it's done.